From e49c9eda3997e5c5a820a230b18c75315d38bcf1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Wed, 13 Sep 2017 18:36:28 -0400
Subject: [PATCH] Add pythonpath to template for bootstrapping.

This package is a dependency of the ros_workspace package which creates
and installs the environment scripts normally used.
---
 debian/rules.em | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules.em b/debian/rules.em
index 9c9604d..5d7d1ef 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -18,6 +18,9 @@ export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 # 	https://github.com/ros-infrastructure/bloom/issues/327
 export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
 
+# Needed to bootstrap since the ros_workspace package does not yet exist.
+export PYTHONPATH=@(InstallationPrefix)/lib/python3.5/site-packages
+
 %:
 	dh $@@ -v --buildsystem=cmake
 
-- 
2.7.4

